src/Utility/DspParams.cpp

```diff
@@ -212,25 +210,27 @@ void DspParams::initIntParams()
 	/** minimum number of processes to wait at the master */
 	IntParams_.createParam("DD/MIN_PROCS", 1);
 
+#ifdef DSP_HAS_CPX
+	IntParams_.createParam("BD/SUB/SOLVER", OsiCpx);
+	IntParams_.createParam("DD/MASTER/SOLVER", OsiCpx);
+	IntParams_.createParam("DD/SUB/SOLVER", OsiCpx);
+	//IntParams_.createParam("DE/SOLVER", OsiCpx);
+	IntParams_.createParam("DW/MASTER/SOLVER", OsiCpx);
+	IntParams_.createParam("DW/SUB/SOLVER", OsiCpx);
+#else
+	IntParams_.createParam("BD/SUB/SOLVER", OsiClp);
 	IntParams_.createParam("DD/MASTER/SOLVER", OsiClp);
 	IntParams_.createParam("DW/MASTER/SOLVER", OsiClp);
-	IntParams_.createParam("BD/SUB/SOLVER", OsiClp);
-
 #ifdef DSP_HAS_SCIP
 	IntParams_.createParam("DD/SUB/SOLVER", OsiScip);
 	IntParams_.createParam("DE/SOLVER", OsiScip);
 	IntParams_.createParam("DW/SUB/SOLVER", OsiScip);
 #endif
-
-#ifdef DSP_HAS_CPX
-	IntParams_.createParam("BD/SUB/SOLVER", OsiCpx);
-	IntParams_.createParam("DD/MASTER/SOLVER", OsiCpx);
-	IntParams_.createParam("DD/SUB/SOLVER", OsiCpx);
-	IntParams_.createParam("DE/SOLVER", OsiCpx);
-	IntParams_.createParam("DW/MASTER/SOLVER", OsiCpx);
-	IntParams_.createParam("DW/SUB/SOLVER", OsiCpx);
 #endif
 
+#ifdef DSP_HAS_GRB
+	IntParams_.createParam("DE/SOLVER", OsiGrb);
```

Comment:
You should have received a warning message if you have gruobi and other solver, because the same parameter is defined twice.